core/setup.py

```diff
@@ -30,7 +32,19 @@ def main():
             [console_scripts]
             autocrypt=autocrypt.cmdline:autocrypt_main
         ''',
-        install_requires = ["click>=6.0", "six"],
+        install_requires=[
+            "click>=6.0",
+             "six"
+        ],
+        dependency_links=[
+            "https://github.com/SecurityInnovation/PGPy.git@\
+             @release/0.4.1#egg=PGPy-0.4.1"
```

Comment:
I use dependency_links to create debian packages.
Re. the release, same comment as above.